packages/js/email-editor/src/private-apis/index.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * WordPress dependencies
+ */
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+import {
+	// @ts-expect-error No types for privateApis.
+	privateApis as blockEditorPrivateApi,
+	// @ts-expect-error No types for privateApis.
+	privateApis as blockEditorPrivateApis,
+} from '@wordpress/block-editor';
+import { privateApis as componentsPrivateApis } from '@wordpress/components';
+import { store as coreStore } from '@wordpress/core-data';
+
+const { unlock } = __dangerousOptInToUnstableAPIsOnlyForCoreModules(
+	'I acknowledge private features are not for use in themes or plugins and doing so will break in the next version of WordPress.',
+	'@wordpress/edit-post' // The module name must be in the list of allowed, so for now I used the package name of the post editor
+);
```

Comment:
I noticed you switched from using the import [here](https://github.com/mailpoet/mailpoet/blob/e728e74d1166a5e572e90aee105a829d4ba0bc30/packages/js/email-editor/src/private-apis/lock-unlock.js). Instead you redefined the same variable. May I ask why that is so? Won't it be better to have a single unlock interface?

Reply:
In that case, can you delete this file https://github.com/mailpoet/mailpoet/blob/e427d17cf3ba4df6ec79e862de07bee078ff5a5c/packages/js/email-editor/src/private-apis/lock-unlock.js

Reply:
Aha. I see I have 2 now 🤦

Reply:
Thank you for noticing this. I missed that one.